packages/mdc-select/foundation.js

```diff
@@ -174,10 +196,12 @@ export default class MDCSelectFoundation extends MDCFoundation {
 
     let maxTextLength = 0;
     for (let i = 0, l = this.adapter_.getNumberOfOptions(); i < l; i++) {
+      const selectBoxAddedPadding = 26;
```

Comment:
Also, given that we're using `getComputedStyle` in this function already, is it feasible to use that here rather than risk skew between SCSS and JS?